packages/api/cms-api/src/generator/crud-generator.decorator.ts

```diff
@@ -21,16 +21,17 @@ export function CrudSingleGenerator(options: CrudSingleGeneratorOptions): ClassD
 }
 
 export interface CrudFieldOptions {
+    output?: boolean; //only for relations, for others customize using @Field
```

Comment:
I'm not sure about this naming, `output` isn't very "GraphQL-y"? Wouldn't it be better to use `fieldResolver` or `resolveField` (would be in line with Nest)?